pyro/distributions/torch_distribution.py

```diff
@@ -78,7 +81,8 @@ def shape(self, sample_shape=torch.Size()):
     def reshape(self, sample_shape=torch.Size(), extra_event_dims=0):
         """
         Reshapes a distribution by adding ``sample_shape`` to its total shape
-        and adding ``extra_event_dims`` to its ``event_shape``.
+        and adding ``extra_event_dims`` to its
+        :attr:`~torch.distributions.distribution.Distribution.event_shape`.
```

Comment:
Hi @fritzo, good to know that we have :func: :attr: to make docs better! Just wonder what is the role of `~`? ^^!

Reply:
The tilde says to print only the last part `event_shape` instead of the full `torch.distributions.distribution.Distribution.event_shape`. I've used this for a number of long links.

Reply:
Got it, thanks!